src/test/java/com/cloudbees/jenkins/plugins/awscredentials/ConfigurationAsCodeTest.java

```diff
@@ -0,0 +1,44 @@
+package com.cloudbees.jenkins.plugins.awscredentials;
+
+import com.cloudbees.plugins.credentials.CredentialsMatchers;
+import com.cloudbees.plugins.credentials.CredentialsProvider;
+import com.cloudbees.plugins.credentials.CredentialsScope;
+import com.cloudbees.plugins.credentials.domains.DomainRequirement;
+import hudson.security.ACL;
+import io.jenkins.plugins.casc.misc.RoundTripAbstractTest;
+import org.jvnet.hudson.test.RestartableJenkinsRule;
+
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.notNullValue;
+import static org.junit.Assume.assumeThat;
+
+public class ConfigurationAsCodeTest extends RoundTripAbstractTest {
+
+    @Override
+    protected void assertConfiguredAsExpected(RestartableJenkinsRule restartableJenkinsRule, String s) {
+        AWSCredentialsImpl credentials = CredentialsMatchers.firstOrNull(
+                CredentialsProvider.lookupCredentials(AWSCredentialsImpl.class,
+                        r.j.jenkins, ACL.SYSTEM, (DomainRequirement) null),
+                CredentialsMatchers.withId("aws-credentials-casc"));
+        assumeThat("AWS credentials are not null", credentials, notNullValue());
+        assumeThat("access key is configured", credentials.getAccessKey(),
+                equalTo("foo"));
+        assumeThat("description is configured", credentials.getDescription(),
```

Comment:
Won't this mark the set as skipped instead of failed?

Reply:
Yes! Thank you for catching that. I recently became fond of `assume` without noticing that it skips the test instead of failing it. Changed to `assert` in https://github.com/jenkinsci/aws-credentials-plugin/pull/77/commits/92108167ef0d3a399d7c1c9befbbc57271784e8b.